workflows/sarek.nf

```diff
@@ -1220,7 +1262,7 @@ def extract_csv(csv_file) {
             def bam         = file(row.bam,   checkIfExists: true)
             def bai         = file(row.bai,   checkIfExists: true)
             def CN          = params.seq_center ? "CN:${params.seq_center}\\t" : ''
-            def read_group  = "\"@RG\\tID:${row.sample}_${row.lane}\\t${CN}PU:${row.lane}\\tSM:${row.sample}\\tLB:${row.sample}\\tPL:${params.seq_platform}\""
+            def read_group  = "\"@RG\\tID:${row.sample}_${row.lane}\\t${CN}PU:${row.lane}\\tSM:${row.patient}_${row.sample}\\tLB:${row.sample}\\tDS:${params.fasta}\\tPL:${params.seq_platform}\""
```

Comment:
I'm wondering why we have row.sample both in the SM-field and in the LB-field? Isn't is enough to have just row.sample in the LB-field?

Reply:
maybe, we should investigate that all callers work then still accordingly.

Reply:
I should learn to keep my mouth shut 😆